Jenkinsfile

```diff
@@ -46,19 +46,34 @@ pipeline {
 		ANDROID_NDK = ""
 
 		PYTHONUNBUFFERED = "true"
+
 		JACOCO_XML = "Paintroid/build/reports/coverage/debug/report.xml"
+		JACOCO_UNIT_XML = "Paintroid/build/reports/jacoco/jacocoTestDebugUnitTestReport/jacocoTestDebugUnitTestReport.xml"
 
 		// place the cobertura xml relative to the source, so that the source can be found
-		COBERTURA_XML = "Paintroid/src/main/java/coverage.xml"
+		JAVA_SRC = "Paintroid/src/main/java"
+
 	}
 
 	options {
 		timeout(time: 2, unit: 'HOURS')
 		timestamps()
-                buildDiscarder(logRotator(numToKeepStr: '30'))
+		buildDiscarder(logRotator(numToKeepStr: '30'))
```

Comment:
NO sarcasm: what a relief, thanks for fixing the inconsistency!

Reply:
Vim points out to me if there is an inconsistency. ;)